media/js/mozorg/home/home-b.js

```diff
@@ -0,0 +1,110 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, Mozilla, dataLayer) {
+    'use strict';
+
+    var HomePageB = {
```

Comment:
This is totally fine for now, but I see this repeating/sharing of code with the sync/send-tabs pages something we could refactor into a reusable component in the future? :)

Reply:
If this logic lives on (which, to be frank, I hope it *doesn't*), yes, definitely worth refactoring into something more reusable.